### ISSUE TITLE
Install libutempter as embeded package 

### ISSUE THREAD

**Issue by Yaraslaut** [contour-terminal/contour#1109]
### Contour Terminal version

Contour 0.3.12.262 

### Installer source

Github: source code cloned

### Operating System

RHEL 8.7

### Architecture

x86-64

### Steps to reproduce

At the moment libutemper can be installed only with package manager of OS (see scripts/install-deps.sh script).
And since it does not have CMakeLists.txt, simple include it into fetch_and_unpack procedure for embeded packages is not possible

### Actual Behavior

Compilation fails on system with no libutempter installed through package manager

**Comment by Yaraslaut** [comment]
@christianparpart  maybe we can make libutemper optional for compilation?

**Comment by christianparpart** [comment]
oh we certainly can make it optional. didn't think about any Linux (!) system NOT having that installed, to be honest. Did you actually encounter that on any of your systems? Which one?

**Comment by uspasojevic96** [comment]
Arch doesn't by default I believe

**Comment by christianparpart** [comment]
Yup. But I thought I updated all install-deps  respectively. I may have missed something? Hmmm

**Comment by Yaraslaut** [comment]
> oh we certainly can make it optional. didn't think about any Linux (!) system NOT having that installed, to be honest. Did you actually encounter that on any of your systems? Which one?

I have rhel with no root, so I would prefer to fetch dependencies as embed as well
